sdk/react/controlBar.js

```diff
@@ -50,19 +50,26 @@ class ControlBar extends Component {
   };
 
   getPlayHeadTimeString = () => {
-    if (this.props.live) {
-      return this.props.live.label;
-    } else {
-      return (Utils.secondsToString(this.props.playhead) + ' - ');
+    const {
+      live, playhead,
+    } = this.props;
+    if (live) {
+      return live.label;
     }
+    return (`${Utils.secondsToString(playhead)} - `);
   };
 
   getDurationString = () => {
-    if (this.props.live) {
-      return null;
-    } else {
-      return Utils.secondsToString(this.props.duration);
+    const {
+      live, duration, playhead,
+    } = this.props;
+    if (live) {
+      if (live.isLive) {
+        return null;
+      }
+      return (` -${Utils.secondsToString(duration - playhead)}`);
```

Comment:
Should that be just `Utils.secondsToString(duration - playhead)`?

Reply:
Probably you are right , but change it to Utils.secondsToString(playhead - duration)

Reply:
After some checking, that code add two symbols at start of string : space and minus, don't see we could improve that place somehow